### PR TITLE
RK-6187 - Reoredered and cleaned jvm setup

### PR DESCRIPTION
**Pull request by LiranLast**

### CODE REVIEW

File: docs/jvm-setup.md [hunk 1]
```diff
@@ -127,6 +127,22 @@ jar {
...
+
+Example
+```
+ROOKOUT_SOURCES=https://github.com/Rookout/Rookout#afe123;/path/to/lib.jar
+```
+
+To load source information from a jar file, you need to add the following attributes to the JAR manifest:
+
+`Rookout-Repository`: repository URL
+`Rookout-Commit`: commit identifier
```
Comment (BenTheKlein) [inline docs/jvm-setup.md:144]: @LiranLast are we sure it Rookout-Commit and not revision?

Comment (LiranLast) [reply docs/jvm-setup.md:144]: I didn't change anything. And you checked the code :)

Comment (BenTheKlein) [reply docs/jvm-setup.md:144]: The code is confusing........

Comment (LiranLast) [reply docs/jvm-setup.md:144]: So let's make it better :)

**Pull request merged**